assets/dag-index-html/README.md

```diff
@@ -0,0 +1,3 @@
+# dag-index-html
+
```

Comment:
Speaking of markdown, @hacdias mind adding release notes stub (similar to one at https://github.com/ipfs/kubo/blob/master/docs/changelogs/v0.17.md, just create v0.18.md), informing users about CBOR/JSON support?

Reply:
done!